### PR TITLE
/gt hand improvements

### PR DESCRIPTION
**Pull request by brachy84**
Now shows material and ore prefix of material items.
Shows fluids in containers on a extra line & add click copy for fluid

### CODE REVIEW

File: src/main/java/gregtech/common/command/CommandHand.java [hunk 1]
```diff
@@ -71,35 +74,49 @@ public void execute(@Nonnull MinecraftServer server, @Nonnull ICommandSender sen
...
-                    ClickEvent fluidClickEvent = new ClickEvent(Action.OPEN_URL, fluidName);
                     player.sendMessage(new TextComponentTranslation("gregtech.command.util.hand.fluid",
-                            fluidName,
                             contents == null ? 0 : contents.amount,
                             properties.getCapacity(),
-                            Boolean.toString(properties.canFill()), Boolean.toString(properties.canDrain()))
-                            .setStyle(new Style().setClickEvent(fluidClickEvent)));
+                            Boolean.toString(properties.canFill()), Boolean.toString(properties.canDrain())));
+                    if (contents != null) {
+                        player.sendMessage(new TextComponentString("Fluid Id: ").appendSibling(new TextComponentString(fluidName).setStyle(new Style().setColor(TextFormatting.GREEN)))
```
Comment (Rongmario) [inline src/main/java/gregtech/common/command/CommandHand.java:82]: Should this be a translation key?

Comment (brachy84) [reply src/main/java/gregtech/common/command/CommandHand.java:82]: Feel free to do it. I'm too lazy

Comment (Rongmario) [reply src/main/java/gregtech/common/command/CommandHand.java:82]: Too bad, I'm also too lazy.

**Comment by Rongmario** [comment]
Conflicting files btw

**Review by Rongmario — APPROVED** [review]

File: src/main/java/gregtech/api/util/CTRecipeHelper.java [hunk 2]
```diff
@@ -43,7 +46,46 @@ public static String getMetaItemId(ItemStack item) {
                 return ((BlockMaterialPipe<?, ?, ?>) block).getPrefix().name + ((BlockMaterialPipe<?, ?, ?>) block).getItemMaterial(item).toCamelCaseString();
             }
         }
+        return null;
+    }
 
+    @Nullable
+    public static Material getMaterial(ItemStack item) {
```
Comment (ALongStringOfNumbers) [inline src/main/java/gregtech/api/util/CTRecipeHelper.java:53]: Why not use `OreDictUnifier#getMaterial(ItemStack)`?

File: src/main/java/gregtech/api/util/CTRecipeHelper.java [hunk 3]
```diff
@@ -43,7 +46,46 @@ public static String getMetaItemId(ItemStack item) {
...
+            }
+            if (block instanceof BlockMaterialPipe) {
+                return ((BlockMaterialPipe<?, ?, ?>) block).getItemMaterial(item);
+            }
+        }
+        return null;
+    }
+
+    @Nullable
+    public static OrePrefix getOrePrefix(ItemStack item) {
```
Comment (ALongStringOfNumbers) [inline src/main/java/gregtech/api/util/CTRecipeHelper.java:73]: Why not use `OreDictUnifier#getPrefix(ItemStack)`?

File: src/main/java/gregtech/api/util/CTRecipeHelper.java (hunk 2 above)
Comment (brachy84) [reply src/main/java/gregtech/api/util/CTRecipeHelper.java:53]: Because i didn't know it exists

**Pull request merged**